src/classes/Auth/Auth.ts

```diff
@@ -17,11 +16,12 @@ import { CreateMagicLinkArgs, MagicLinkOptions } from './types';
 export class Auth extends PassageBase {
     private readonly jwks: (protectedHeader?: JWSHeaderParameters, token?: FlattenedJWSInput) => Promise<KeyLike>;
     private readonly magicLinksApi: MagicLinksApi;
+
     /**
      * Auth class constructor.
      * @param {PassageInstanceConfig} config config properties for Passage instance
      */
-    public constructor(protected config: PassageInstanceConfig) {
+    public constructor(protected readonly config: PassageInstanceConfig) {
```

Comment:
You can also mark the constructor in `PassageBase` as `protected readonly` which should enforce that for inherited classes

Reply:
updated in [8733ffb](https://github.com/passageidentity/passage-node/pull/209/commits/8733ffb8bc0374542dca4ea1529b442b5f804cb8)